src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,55 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.exception.Messages;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a done user command.
+ * Inherits from command class.
+ */
+public class DoneCommand extends Command {
```

Comment:
Just to add from Marcus's Suggestion maybe consider a class CommandManager to manage all the commands and have an enum of different types of command instead?